videos/models.py

```diff
@@ -21,9 +25,25 @@ class Video(models.Model):
     moderated_by = models.ForeignKey('users.User', null=True, blank=True)
     moderation_notes = models.TextField(null=True, blank=True)
     length = models.IntegerField(null=True, blank=True)
+    tags = TaggableManager(blank=True)
 
     def __str__(self):
         return ''.join(('Video: ', self.title, ' ', self.uploader_name))
 
     def get_absolute_url(self):
         return reverse('api:video-detail', kwargs={'pk': self.pk})
+
+    @property
+    def score_avg(self):
+        """
+
+        :return: Decimal of the Average Score for the Video
+        """
+        return self.videoscore_set.aggregate(Avg('score')).get('score__avg')
+
+
+class VideoScore(models.Model):
```

Comment:
The users in the "jury" will score the video, then the highest scoring ones
have the greater chance of getting into the show. Depending on submissions,
all videos may make it in, but this allows them to rank the videos.

On Thu, Jul 21, 2016, 1:20 PM Brian S. Corbin <notifications@github.com>
wrote:

> In videos/models.py
> <https://github.com/hub-ology/video_village/pull/6#discussion_r71747920>:
>
> >
> >      def __str__(self):
> >          return ''.join(('Video: ', self.title, ' ', self.uploader_name))
> >
> >      def get_absolute_url(self):
> >          return reverse('api:video-detail', kwargs={'pk': self.pk})
> > +
> > +    @property
> > +    def score_avg(self):
> > +        """
> > +
> > +        :return: Decimal of the Average Score for the Video
> > +        """
> > +        return self.videoscore_set.aggregate(Avg('score')).get('score__avg')
> > +
> > +
> > +class VideoScore(models.Model):
>
> Is a VideoScore primarily used for folks to "vote" on videos and then the
> highest scoring videos get into the shows?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hub-ology/video_village/pull/6/files/a83d0b3cf9928a374fe5c99cab51701b23ad77f5#r71747920>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABR8x6cKAa0RycxMl58IlMLVCMfSGR3vks5qX6pfgaJpZM4JSBX_>
> .
>